spec/lib/data_update_scripts/add_single_resource_role_to_tech_admins_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "rails_helper"
+require Rails.root.join(
+  "lib/data_update_scripts/20210209185037_add_single_resource_role_to_tech_admins.rb",
+)
+
+describe DataUpdateScripts::AddSingleResourceRoleToTechAdmins do
```

Comment:
This is the exact same spec as `spec/lib/data_update_scripts/add_single_resource_admin_role_to_users_with_tech_admin_spec.rb` from my previous commit, but renamed to fit the new DataUpdateScript